app/templates/share.js

```diff
@@ -10,20 +10,61 @@ function passwordComplete(state, password) {
   const el = html([
     `<div class="selectPassword">${state.translate('passwordResult', {
       password: '<pre></pre>'
-    })}</div>`
+    })}
+    <button id="resetButton">Reset</button>
```

Comment:
I missed this earlier... The "Reset" text here needs to be localized. We need to add a string to `public/locales/en-US/send.ftl` and use `state.translate` like the other strings here. I also think "Change" is a better word than "Reset" in this case.

Reply:
Yes , I know that it has to be localised. But I thought I would have to do it in a number of different languages . Thus I had left it alone. I'll make the changes soon.